src/Compilers/Core/CodeAnalysisTest/MetadataReferences/MetadataReferenceTests.cs

```diff
@@ -504,19 +503,6 @@ public void Equivalence()
             }
         }
 
-        [Fact]
```

Comment:
Why was this test deleted?

Reply:
It does not test anything. Half of it is unused code.